scripts/zones/Rolanberry_Fields/IDs.lua

```diff
@@ -65,9 +65,11 @@ zones[xi.zone.ROLANBERRY_FIELDS] =
             [17228149] = 17228150, -- 395 -24 -147
         },
 
-        SILK_CATERPILLAR  = 17227782,
-        SIMURGH           = 17228242,
-        CHUGLIX_BERRYPAWS = 17228249,
+        SILK_CATERPILLAR     = 17227782,
+        SIMURGH              = 17228242,
+        CHUGLIX_BERRYPAWS    = 17228249,
+        BLACK_TRIPLE_STARS_1 = 17227972,
```

Comment:
he's a lotto, he just can't pop/will depop if its day but otherwise lotto.

```
17227968    Midnight Wings
17227969    Midnight Wings
17227970    Midnight Wings
17227971    Midnight Wings
17227972    Black Triple Stars
```
```
17227988    Midnight Wings
17227989    Midnight Wings
17227990    Midnight Wings
17227991    Midnight Wings
17227992    Black Triple Stars
```
One midnight wings in each group is the PH for its respective Triple stars. Only 2 of the NMs 2 copies is up at once, just like Leaping Lizzy.

Reply:
Sorry, I'm confused what you're saying here. I have those listed there to use in the zone.lua to prevent it from spawning during the day period so the respawn timer doesn't get triggered if it spawns during the day and immediately gets despawned.

Reply:
I was confused because he was already having
```lua
    mob =
    {
        BLACK_TRIPLE_STARS_PH =
        {
            [17227968] = 17227972, -- 4 -16 -119 (north)
            [17227988] = 17227992, -- 76 -15 -209 (south)
        },
```
so there's 2 sets of entries for his IDs now, at the time I 1st posted I thought these were missing. 

Now that I realize it though, it should be possible to get the IDs from this suitable though rather than defining 2 more entries tho that is kinda nested deep, so I am going to leave that for another reviewer to decide if its fine as is..

Reply:
tl;dr teo was being a lil slow

Reply:
Could you let me know how you would reference the NM's ID in the table? I would much prefer to do it that way, just not sure how to.

Reply:
I think yer gonna have to leave it for now coz I can't for the life of me get the syntax correct I'm sorry xD